nsdb-kafka-connect/src/main/scala/io/radicalbit/nsdb/connector/kafka/sink/NsdbSinkWriter.scala

```diff
@@ -35,6 +36,9 @@ class NsdbSinkWriter(connection: NSDB,
                      globalNamespace: Option[String],
                      defaultValue: Option[String])
     extends StrictLogging {
+
+  implicit val loggerImpl: Logger = logger
```

Comment:
can we avoid to use an implicit for logging purposes?

Reply:
Ok, I'll give a shot by employing what here mentioned https://stackoverflow.com/questions/35363928/scala-use-same-logger-into-class-and-companion-object

Reply:
Ok, I moved to placeholders, I was confused by scala_logging employment here. My fault.